Classes/Emogrifier.php

```diff
@@ -671,6 +671,7 @@ private function copyCssWithMediaToStyleNode(\DOMDocument $xmlDocument, \DOMXPat
             foreach ($this->parseSelectors($mediaQuery['css']) as $selector) {
                 if ($this->existsMatchForCssSelector($xpath, $selector['selector'])) {
                     $mediaQueriesRelevantForDocument[] = $mediaQuery['query'];
+                    break;
```

Comment:
Why this change? Is it necessary for fixing the bug?

(As far as I can see, it only limits the media queries to exactly one query.)